argh_derive/src/parse_attrs.rs

```diff
@@ -268,6 +268,7 @@ pub struct TypeAttrs {
     pub is_subcommand: Option<syn::Ident>,
     pub name: Option<syn::LitStr>,
     pub description: Option<Description>,
+	pub disable_help: Option<syn::LitBool>,
```

Comment:
Stylistically, I think flipping this to "enable_help" reads a bit better. That avoids devs having to mentally parse expressions like `if !disable_help { ... }`.

Reply:
I chose `disable_help` because as a dev I usually assume optional boolean flags to default to `false`, but I can change that if you want